packages/flutter/lib/src/services/clipboard.dart

```diff
@@ -14,7 +14,7 @@ import 'system_channels.dart';
 @immutable
 class ClipboardData {
   /// Creates data for the system clipboard.
-  const ClipboardData({ this.text });
+  const ClipboardData({ required String this.text });
 
   /// Plain text variant of this clipboard data.
   final String? text;
```

Comment:
Maybe consider leaving an implementation comment here explaining why this is nullable even though currently it is always guaranteed to be non-null?

Reply:
Added!